packages/cozy-konnector-libs/src/libs/hydrateAndFilter.js

```diff
@@ -133,7 +133,7 @@ const hydrateAndFilter = (documents, doctype, options = {}) => {
   }
 
   const defaultShouldSave = () => true
-  const defaultShouldUpdate = existing => false
+  const defaultShouldUpdate = existing => false // eslint-disable-line no-unused-vars
```

Comment:
mmmhhh, thank you lord eslint!
But it seems that the variable should be a function that always returns `false' regardless of the given parameter.

So why do we have a parameter?

Reply:
I chose to still display this parameter to an eventual user of the shouldUpdate option that this parameter is available.